erpnext/setup/doctype/email_digest/email_digest.py

```diff
@@ -825,4 +826,14 @@ def get_count_for_period(account, fieldname, from_date, to_date):
 		last_year_closing_count = get_count_on(account, fieldname, fy_start_date - timedelta(days=1))
 		count = count_on_to_date + (last_year_closing_count - count_before_from_date)
 
-	return count
\ No newline at end of file
+	return count
+
+def get_future_date_for_calendaer_event(frequency):
+	from_date = to_date = now_datetime().date()
+
+	if frequency == "Weekly":
+		to_date = now_datetime().date() - relativedelta(days=1) + relativedelta(weeks=1)
+	elif frequency == "Monthly":
+		to_date = now_datetime().date() - relativedelta(days=1) + relativedelta(months=1)
```

Comment:
Use `add_days` and `add_months` functions

Reply:
Fixed